index.html

```diff
@@ -650,6 +654,20 @@ <h3>Badgeyay</h3>
 					</div>
 
 
+         	<div class="col-md-4 col-sm-6">
+						<div class="info-box">
+							<img alt="WebsiteGenerator" src="img/websitegen.png">
```

Comment:
Update the image

Reply:
@uds5501  Thanks for the review . I will make the changes.